lib/parallel_tests/tasks.rb

```diff
@@ -125,6 +128,20 @@ def build_run_command(type, args)
         command
       end
 
+      def configured_databases
+        return [] unless defined?(ActiveRecord) && rails_61_or_greater?
+
+        @@configured_databases ||= ActiveRecord::Tasks::DatabaseTasks.setup_initial_database_yaml
+      end
+
+      def for_each_database(&block)
+        return unless configured_databases.present? && block_given?
+
```

Comment:
can just remove this ?

Reply:
Removing the `configured_databases.present?` piece is fine since it'll just be empty otherwise 😄 Since this is an internal API, if you're comfortable not having the safety of `block_given?` then I can yoink that out as well!

Reply:
👍